docs/core/setup/install-sdk.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Install .NET Core SDK on Windows, Linux, and macOS
+description: Learn how to install .NET Core on Windows, Linux, and macOS. Discover the dependencies required to develop .NET Core apps.
+author: thraka
+ms.author: adegeo
+ms.date: 11/06/2019
+zone_pivot_groups: operating-systems-set-one
+---
+
+# Install the .NET Core SDK
+
+In this article you'll learn how to download and install the .NET Core SDK. The .NET Core SDK is used to create create .NET Core apps and libraries. The .NET Core runtime is always installed with the SDK.
+
+Here are links to the .NET Core downloads page by version:
+
+- [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+- [.NET Core 2.2 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.2).
+- [.NET Core 2.1 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.1).
+
+## Install with an IDE
+
+When you install a supported IDE, you can also install .NET Core. By keeping the IDE up-to-date, you'll receive the latest supported .NET Core SDK.
+
+::: zone pivot="os-windows"
+
+- Download [Visual Studio](https://www.visualstudio.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2019)\
```

Comment:
you need to do more than just install VS to get the SDK installed. This is only explained in the other section. Why this is not combined?